specification/billing/resource-manager/readme.md

```diff
@@ -28,6 +28,14 @@ These are the global settings for the Billing API.
 openapi-type: arm
 tag: package-2020-05
 ```
+### Tag: package-2020-11-privatepreview
+
+These settings apply only when `--tag=package-2020-11-privatepreview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-11-privatepreview'
```

Comment:
Private preview API should not be documented in Public repo. Make this change in private repo.